Quadriga/src/test/java/edu/asu/spring/quadriga/service/publicwebsite/impl/AboutTextManagerTest.java

```diff
@@ -0,0 +1,67 @@
+package edu.asu.spring.quadriga.service.publicwebsite.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import edu.asu.spring.quadriga.dao.impl.publicwebsite.AboutTextDAO;
+import edu.asu.spring.quadriga.dto.AboutTextDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.impl.publicwebsite.AboutTextManager;
+
+public class AboutTextManagerTest {
+
+    @InjectMocks
+    private AboutTextManager aboutTextManager;
+
+    @Mock
+    private AboutTextDAO mockedAboutTextDAO;
```

Comment:
I tried using interface, but I don't understand the errors I get from jUnit. This is what I am trying:
 @Mock
    private IAboutTextDAO mockedAboutTextDAO;
 
@Before
    public void setUp() throws QuadrigaStorageException {
        MockitoAnnotations.initMocks(this);
        mockedAboutTextDAO = Mockito.mock(AboutTextDAO.class);
     }
It gives the following error:
Wanted but not invoked. I think that means that calling saveAbout() was not executed properly when I am using interface and telling mockito to use AboutTextDAO class.